# contracts/interfaces/IFHERC20.sol  ->  contracts/interfaces/IFHERC20.fsol
# upstream 5138cb8 .. ported 86f1b7c

diff --git a/contracts/interfaces/IFHERC20.sol b/contracts/interfaces/IFHERC20.fsol
index 9c95e40..fd782c0 100644
--- a/contracts/interfaces/IFHERC20.sol
+++ b/contracts/interfaces/IFHERC20.fsol
@@ -3,7 +3,7 @@ pragma solidity ^0.8.25;
 
 import { euint64 } from "@fhenixprotocol/cofhe-contracts/FHE.sol";
 import { IERC20 } from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
-import { IERC7984 } from "./IERC7984.sol";
+import { IERC7984 } from "./IERC7984.fsol";
 
 /**
  * @dev Interface for a confidential fungible token standard utilizing the Fhenix FHE library.
